designer/server/src/routes/forms/create.js

```diff
@@ -13,6 +13,15 @@ import { buildErrorDetails } from '~/src/common/helpers/build-error-details.js'
 import * as forms from '~/src/lib/forms.js'
 import * as create from '~/src/models/forms/create.js'
 
+/**
+ * Gets the author from the auth credentials
+ * @param {import('@hapi/hapi').UserCredentials | undefined} user
+ */
+function getAuthor(user) {
+  // @ts-expect-error - user is not undefined
```

Comment:
Sorry Dave, we've said above that `user` might be `undefined`

Don't need this function if we do the extra check from https://github.com/DEFRA/forms-designer/pull/148#discussion_r1596733832

```suggestion
  if (!user) {
    return
  }

```